scss/button/_theme.scss

```diff
@@ -2,7 +2,7 @@
 $button-focused-box-shadow: 0 3px 4px 0 rgba(0, 0, 0, .06) !default;
 $button-active-box-shadow: inset 0 2px 2px 0 rgba(0, 0, 0, .06) !default;
 
-$primary-focused-box-shadow: 0 3px 4px 0 rgba($accent, .06) !default;
+$primary-focused-box-shadow: 0 3px 4px 0 rgba($accent, .4) !default;
```

Comment:
Variables must be placed in the variables file: https://github.com/telerik/kendo-theme-default/blob/master/scss/_variables.scss#L173